telegram/src/main/kotlin/com/github/kotlintelegrambot/entities/Location.kt

```diff
@@ -2,5 +2,6 @@ package com.github.kotlintelegrambot.entities
 
 data class Location(
     val longitude: Float,
-    val latitude: Float
+    val latitude: Float,
+    val horizontal_accuracy: Float?,
```

Comment:
Use camel case like in other variables. In case you need to convert the field there is an example here https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/pull/246/files#diff-a2f2723c9a12ea5c753ed62dc6c8c7f17869bd7e1c949b298ee726daa43ca9e7R8